tools/automatic_query_fixer/src/test/java/com/google/cloud/bigquery/utils/queryfixer/QueryTokenProcessorTest.java

```diff
@@ -0,0 +1,30 @@
+package com.google.cloud.bigquery.utils.queryfixer;
```

Comment:
nit: not sure why there are so many empty lines in between...

Reply:
I think it is caused by auto format.